09_proxmox_fstrim_cloudinit_ansible_lvm_08-sep-2022/cloudinit-exp/cloudinit.md

```diff
@@ -16,7 +17,7 @@ cd ./cloudinit-exp/
 nano ./create-rocky9-dc0.sh
```

Comment:
please avoid using _I_ or _my_ at all.

I have this script I've passed from another vm via scp, it is placed here -> This script passed from another VM via scp, it is placed here:

(please change vm to VM and vms to VMs throughout the files).